src/VisualStudio/CodeLens/ReferenceCodeLensProvider.cs

```diff
@@ -0,0 +1,280 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeLens;
+using Microsoft.CodeAnalysis.Editor;
+using Microsoft.CodeAnalysis.Editor.Wpf;
+using Microsoft.CodeAnalysis.Remote;
+using Microsoft.ServiceHub.Client;
+using Microsoft.VisualStudio.Core.Imaging;
+using Microsoft.VisualStudio.Language.CodeLens.Remoting;
+using Microsoft.VisualStudio.Language.Intellisense;
+using Microsoft.VisualStudio.LanguageServices.CodeLens;
+using Microsoft.VisualStudio.LanguageServices.Remote;
+using Microsoft.VisualStudio.Threading;
+using Microsoft.VisualStudio.Utilities;
+using Roslyn.Utilities;
+using StreamJsonRpc;
+
+namespace Microsoft.VisualStudio.CodeAnalysis.CodeLens
+{
+    [Export(typeof(IAsyncCodeLensDataPointProvider))]
+    [Name(Id)]
+    [ContentType(ContentTypeNames.CSharpContentType)]
+    [ContentType(ContentTypeNames.VisualBasicContentType)]
+    [LocalizedName(typeof(CodeLensVSResources), "References")]
```

Comment:
I don't see a matching resource named `"References"` above. Perhaps you meant to use `"CSharp_VisualBasic_References"`?

Reply:
thank you